src/main/flight/pid.c

```diff
@@ -205,21 +209,18 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
     static float previousRateError[2];
     static float previousSetpoint[3];
 
-    if(FLIGHT_MODE(HORIZON_MODE))
-        calcHorizonLevelStrength(pidProfile);
```

Comment:
It's a deliberate choice. The question is "who pays"?

In the old code when flying in acro mode there was a test `if(FLIGHT_MODE(HORIZON_MODE))` for every pid loop. In my code `calcHorizonLevelStrength` is calculated per axis, but only when in `HORIZON_MODE`.

So acro mode flyers benefit at the expense of horizon mode flyers. That's how it should be for betaflight.